javalib/src/main/scala/java/lang/IEEE754Helpers.scala

```diff
@@ -1,6 +1,7 @@
 package java.lang
 
 import scalanative.unsafe._
+import scalanative.unsigned._
 import scalanative.libc.errno
 
 import scalanative.posix.errno.ERANGE
```

Comment:
Come to think about it, maybe this is better for another PR.

Reply:
I would agree that it might make cleaner commit history if it that change would be applied in other PR. Especially if it would be needed to make some additional changes or change dependencies in `build.sbt`